### ISSUE TITLE
nonesense message in pico_extras_import.cmake

### ISSUE THREAD

**Issue by smithps** [raspberrypi/pico-extras#39]
Line 53 has the message;

`
"PICO EXTRAS location was not specified. Please set PICO_EXTRAS_PATH or set PICO_EXTRAS_FETCH_FROM_GIT to on to fetch from git."
`
should it really be;
 
... PICO_EXTRAS_FETCH_FROM_GIT to fetch from git.

**Comment by lurch** [comment]
It defaults to being 'off', so you have to set it 'on' to make use of that feature? :shrug:  https://github.com/raspberrypi/pico-extras/blob/master/external/pico_extras_import.cmake#L52

**Comment by JamesH65** [comment]
I agree with @smithps, changing to "ON" would make that a lot clearer.

**Comment by lurch** [comment]
Feel free to submit a PR @smithps
